### ISSUE TITLE
bug: Navigation works only once in a nested ion-router-outlet

### ISSUE THREAD

**Issue by guzmanoj** [ionic-team/ionic#20991]
**Ionic version:**
[x] **5.0.7**

**Current behavior:**
Given a set of navigation items, eg:
```
<a [routerLink]="['publish']">To Publish</a>
<a [routerLink]="['management']">To Management</a>
```
When clicking on the items the navigation is triggered only the first time the user clicks.

**Expected behavior:**
By clicking on the items, the URL should change and the user shall be able to view the content change accordingly due to the redirection.

**Steps to reproduce:**
Please check the provided Stackblitz.

**Related code:**
https://stackblitz.com/edit/ionic-4-child-routes-navigation-issue
(Manually point the URL to /cars/123)

**Ionic info:**

```
Ionic:

   Ionic CLI                     : 6.4.3 (/usr/local/lib/node_modules/@ionic/cli)
   Ionic Framework               : @ionic/angular 5.0.7
   @angular-devkit/build-angular : 0.901.0
   @angular-devkit/schematics    : 9.1.0
   @angular/cli                  : 9.1.0
   @ionic/angular-toolkit        : 2.2.0

Capacitor:

   Capacitor CLI   : 2.0.0
   @capacitor/core : 2.0.0

Utility:

   cordova-res : not installed
   native-run  : not installed

System:

   NodeJS : v12.14.1 (/usr/local/Cellar/node@12/12.14.1_1/bin/node)
   npm    : 6.13.4
   OS     : macOS Catalina
```

**Comment by indraraj26** [comment]
Hi,

You forget to lazyload publish and management feature module in app-routing.module.ts

**Comment by guzmanoj** [comment]
> Hi,
> 
> You forget to lazyload publish and management feature module in app-routing.module.ts

Well, not really. The routes are being lazy loaded within the `shell-routing.module.ts`, is a pattern that actually works (check here https://stackblitz.com/edit/nested-lazy-loaded-routes)
But never the less, let's assume that all routes needed to be placed in the `app-routing.module.ts`, it still doesn't work as expected.

**Comment by liamdebeasi** [comment]
Thanks for the issue. I am going to close this as a duplicate of https://github.com/ionic-team/ionic/issues/18197.
